dask/dataframe/io/csv.py

```diff
@@ -517,6 +517,10 @@ def read_pandas(
             2
         ]
 
+        # Check for at least one valid path
+        if len(paths) == 0:
+            raise OSError("%s resolved to no files" % urlpath)
```

Comment:
I just realized that we'd like to have Python-3 optimization wherever we can in the codebase, so we may want to use f-strings here? It's just a nice-to-have, so completely up to you. :)

Reply:
I honestly use f-strings way more, I'm definitely fine to change this. Wanted to keep consistency with existing code, but if we're trying to use f-strings more I'll change it.